common/app/common/InlineStyles.scala

```diff
@@ -86,7 +86,8 @@ object InlineStyles {
     // Outlook ignores styles with !important so we need to strip that out.
     // So this doesn't change which styles take effect, we also sort styles
     // so that all important styles appear to the right of all non-important styles.
-    document.getAllElements.asScala.foreach { el =>
+    document.getAllElements.asScala.filter(el => el.attr("style") != "")
+      .foreach { el =>
```

Comment:
indentation is a bit weird here

Reply:
Happy to make it more Scala-idiomatic. Would you put it all on one line?